For various reasons sometimes the debcargo-conf patches can't be applied as-is,
so we further patch them here. The reasons are as follows:

- commoncrypto, commoncrypto-sys: these crates were released before crates.io
  automatically re-wrote all Cargo.toml files. cargo-vendor outputs the
  un-re-written original Cargo.toml files, but debcargo will forcibly rewrite
  even un-re-written ones that were published to crates.io. The patch below
  rewrites our existing patches so they work against the un-re-written ones.

- our patch to miniz_oxide replaces the adler dep for the adler32 dep. This
  interferes with our primitive logic in debian-cargo-vendor, so we have to
  undo that here.

--- a/src/commoncrypto-sys/debian/patches/no-clippy.patch
+++ b/src/commoncrypto-sys/debian/patches/no-clippy.patch
@@ -1,17 +1,16 @@
---- a/Cargo.toml	2017-01-24 06:56:51.000000000 +0000
-+++ b/Cargo.toml	2018-09-15 15:37:47.602333479 +0000
-@@ -19,14 +19,8 @@
+--- a/Cargo.toml
++++ b/Cargo.toml
+@@ -8,13 +8,8 @@
  keywords = ["crypto", "hash", "digest", "osx", "commoncrypto"]
  license = "MIT"
- repository = "https://github.com/malept/rust-commoncrypto"
--[dependencies.clippy]
--version = "0.0"
--optional = true
  
- [dependencies.libc]
- version = "0.2"
- [dev-dependencies.hex]
- version = "0.2"
--
 -[features]
 -lint = ["clippy"]
+-
+ [dependencies]
+ libc = "0.2"
+ 
+-clippy = { version = "0.0", optional = true }
+-
+ [dev-dependencies]
+ hex = "0.2"
--- a/src/commoncrypto/debian/patches/no-clippy.patch
+++ b/src/commoncrypto/debian/patches/no-clippy.patch
@@ -1,17 +1,16 @@
---- a/Cargo.toml	2017-01-24 06:56:51.000000000 +0000
-+++ b/Cargo.toml	2018-09-15 17:37:01.354423224 +0000
-@@ -19,14 +19,8 @@
+--- a/Cargo.toml
++++ b/Cargo.toml
+@@ -8,13 +8,8 @@
  keywords = ["crypto", "hash", "digest", "osx", "commoncrypto"]
  license = "MIT"
- repository = "https://github.com/malept/rust-commoncrypto"
--[dependencies.clippy]
--version = "0.0"
--optional = true
  
- [dependencies.commoncrypto-sys]
- version = "0.2.0"
- [dev-dependencies.hex]
- version = "0.2"
--
 -[features]
 -lint = ["clippy"]
+-
+ [dependencies]
+ commoncrypto-sys = { version = "0.2.0", path = "../commoncrypto-sys" }
+ 
+-clippy = { version = "0.0", optional = true }
+-
+ [dev-dependencies]
+ hex = "0.2"
--- a/src/miniz_oxide/debian/patches/series
+++ b/src/miniz_oxide/debian/patches/series
@@ -1,3 +1,2 @@
 remove-rustc-dep-of-std-etc.patch
 remove-compiler-builtins.patch
-use-adler32.patch
